rust/kcl-lib/src/std/array.rs

```diff
@@ -250,7 +270,7 @@ pub async fn push(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, K
             message: format!("You can't push to a value of type {actual_type}, only an array"),
```

Comment:
Hmm, patterns like this are probably what Ben ran into. I bet that Homogenous Arrays display the same as mixed arrays. He was getting, "Expected an array but found array (list)".